### ISSUE TITLE
Segmented valueObservers not working

### ISSUE THREAD

**Issue by darkwood-studios** [TheJuki/KFormMaster#110]
Version 6.3.3

I've added a segmented elemented which holds some segmented list items. Per default the second item ist active.

The problem is, that the valueObserver is not called at the first time. I have to change the value twice to trigger the value change listener.

here is my example:

```
var itemList = listOf(
        SegmentedListItem(id = 1, name = "test1", drawableRes = R.drawable.ico_strom),
        SegmentedListItem(id = 2, name = "test2", drawableRes = R.drawable.ico_gas),
        SegmentedListItem(id = 3, name = "test3", drawableRes = R.drawable.ico_waerme)
    )

segmented<SegmentedListItem> {
                options = itemList
                value = itemList[1]
                valueObservers.add { newValue, element ->
                    doSomething()
                }
            }
```

```
data class SegmentedListItem(val id: Long? = null, val name: String? = null, @DrawableRes override var drawableRes: Int? = 0) : SegmentedDrawable {
    override fun toString(): String {
        return name.orEmpty()
    }
}
```

**Comment by TheJuki** [comment]
See #108. Should be fixed in 6.3.4.

**Comment by darkwood-studios** [comment]
Ah, great. Works for me. Thanks.
